modules/newborntownWebBidAdapter.js

```diff
@@ -0,0 +1,157 @@
+import * as utils from '../src/utils';
+import {registerBidder} from '../src/adapters/bidderFactory';
+import {BANNER, NATIVE} from '../src/mediaTypes';
+const BIDDER_CODE = 'newborntownWeb';
+
+let useSSL = document.location.protocol == 'https:';
```

Comment:
As a part of prebid-3.0 release (http://prebid.org/blog/pbjs-3), all adapters can call only secure endpoints, so you shall remove this line.